samples/wifi/scan/prj.conf

```diff
@@ -8,28 +8,18 @@ CONFIG_WIFI_NRF700X=y
 CONFIG_NET_L2_WIFI_MGMT=y
 CONFIG_HEAP_MEM_POOL_SIZE=25000
 
-CONFIG_NET_PKT_RX_COUNT=1
-CONFIG_NET_PKT_TX_COUNT=1
-
 # System settings
 CONFIG_NEWLIB_LIBC=y
 CONFIG_NEWLIB_LIBC_NANO=n
```

Comment:
I guess without the supplicant we can get this working with nano, its much smaller form factor.